community/cypher/spec-suite-tools/src/test/scala/cypher/feature/steps/SpecSuiteSteps.scala

```diff
@@ -73,6 +80,13 @@ trait SpecSuiteSteps extends FunSuiteLike with Matchers with TCKCucumberTemplate
     // do nothing, but necessary for the scala match
   }
 
+  private val PENDING = """^this scenario is pending on: (.+)$"""
+  And(PENDING) { (reason: String) =>
```

Comment:
Yes, basically. I added it since I discovered contentious topics during porting the tests; I can't progress without having a decision on the actual CIP. I also don't want to loose the test. I could have kept the scala test instead but then I imagine similar situations could happen in the future so it felt like a good way to be able to write scenarios up that are still being discussed ahead of time.